scripts/requirements-extras.txt

```diff
@@ -19,7 +19,9 @@ Pillow
 imgtool>=1.7.1
 
 # used by nanopb module to generate sources from .proto files
-protobuf
+# Google introduced breaking changes in protobuf v4.21.0, use v3 for now
+# see https://developers.google.com/protocol-buffers/docs/news/2022-05-06
+protobuf<4
```

Comment:
Am I understanding correctly that this prevents the use of v4.* of protobuf since it has a breaking change? Will this go away once the pull request is merged?

Reply:
OK, so I did understand correctly that this is a temporary fix? If so I'm happy with it, can we just make sure there's an issue tracking this so we can fix it up later and make sure that the release team is aware of it (@fabiobaltieri)

Reply:
There was a discussion when the `nanopb` was introduced in zephyr's PR #33211. If `grpcio-tools` works on windows, this might be the current preferred way.
This could also eliminate the requirement to have `protoc` installed in the build system. @carlescufi 

Otherwise we would have to work with this temporary requirement. I opened a [nanopb issue](https://github.com/nanopb/nanopb/issues/787) where the maintainer also suggests to use `grpcio-tools`.